File: docs/source/examples/Schemas/ProjectAndTask.swift
class Task: Object {
    @objc dynamic var name = ""
    @objc dynamic var isComplete = false
    @objc dynamic var assignee: String? = nil
    @objc dynamic var priority = 0;
    @objc dynamic var progressMinutes = 0;
}

class Project: Object {
    @objc dynamic var name = ""
    let tasks = List<Task>();
}

File: docs/source/examples/Schemas/ProjectAndTask.java
// ProjectTask.java
public class ProjectTask extends RealmObject {
    @Required
    public String name;
    public String assignee;
    public int progressMinutes;
    public boolean isComplete;
    public int priority;
    public String _partition;
}

// Project.java
public class Project extends RealmObject {
    @Required
    public String name;
    public RealmList<ProjectTask> tasks = new RealmList<>();
    public String _partition;
}
